src/interfaces/node.h

```diff
@@ -26,11 +26,9 @@ class Coin;
 class RPCTimerInterface;
 class UniValue;
 class proxyType;
-enum class FeeReason;
```

Comment:
Also remove `CCoinControl` forward declaration?

Reply:
Will leave it in for now, because I don't want to go through another rebase-reACK iteration just for this. Also, there are redundant forward declarations in other parts of the code, so this is not making the overall problem worse.

Reply:
Yeah, don't mind that, just noted in case you have to push again.